guides/source/active_record_multiple_databases.md

```diff
@@ -280,10 +280,10 @@ the middleware for automatic switching.
 Automatic switching allows the application to switch from the writer to replica or replica
 to writer based on the HTTP verb and whether there was a recent write by the requesting user.
 
-If the application is receiving a POST, PUT, DELETE, or PATCH request the application will
-automatically write to the writer database. For the specified time after the write, the
-application will read from the primary. For a GET or HEAD request the application will read
-from the replica unless there was a recent write.
+If the application receives a POST, PUT, DELETE, or PATCH request, the application will
+automatically write to the writer database. The application will read from the primary
+for the specified time after the write. The application will read from the replica unless
+there was a recent write for a GET or HEAD request.
```

Comment:
I think we can do even better here. The conditions used in the three sentences all kind of overlap which makes it hard to follow the underlying decision tree. We should try to simplify it so that the logic is clear:

- is the request POST, PUT, DELETE, PATCH? -> writer
- was there a recent write? -> writer
- else -> reader

```suggestion
If the application receives a POST, PUT, DELETE, or PATCH request, the application will
automatically write to the writer database. If the request is not one of those methods,
but the application recently made a write, the writer database will also be used. All
other requests will use the replica database.
```